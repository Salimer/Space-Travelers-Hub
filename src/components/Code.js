import styled from 'styled-components';

const Code = styled.pre`
  white-space: pre-wrap;
  white-space: -moz-pre-wrap;
  white-space: -o-pre-wrap;
  word-wrap: break-word;
`;

export default Code;
