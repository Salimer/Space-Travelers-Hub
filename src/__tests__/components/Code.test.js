import renderer from 'react-test-renderer';
import Code from '../../components/Code';

describe('testing Code component', () => {
  test('render', () => {
    const tree = renderer.create(<Code>{JSON.stringify({ data: { key: 'value' } })}</Code>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
