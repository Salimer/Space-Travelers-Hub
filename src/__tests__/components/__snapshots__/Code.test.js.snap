// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing Code component render 1`] = `
<pre
  className="sc-aXZVg cwbUMI"
>
  {"data":{"key":"value"}}
</pre>
`;
